src/qibosoq/rfsoc_server.py

```diff
@@ -1,20 +1,28 @@
 """Qibosoq server for qibolab-qick integration.
 
-Supports the following FPGA:
+Tested on the following FPGA:
     * RFSoc4x2
+    * ZCU111
 """
 
+import json
 import logging
 import os
-import pickle
 import socket
+import traceback
 from socketserver import BaseRequestHandler, TCPServer
 
 from qick import QickSoc
 
-from qibosoq.qick_programs import ExecutePulseSequence, ExecuteSingleSweep
+from qibosoq.abstracts import Config, Pulse, Qubit, Sweeper
+from qibosoq.qick_programs import ExecutePulseSequence, ExecuteSweeps
 
 logger = logging.getLogger("__name__")
```

Comment:
```suggestion
logger = logging.getLogger(__name__)
```

[`__name__`](https://docs.python.org/3/reference/import.html#__name__) is a special variable.